SPRNetTool/Domain/BitmapDisplayManager.cs

```diff
@@ -353,9 +357,10 @@ private async Task PlayAnimation()
                         }
                     }
                 }
-
+                DisplayedBitmapSourceCache.ColorSourceCaching[frameIndex] = DisplayedBitmapSourceCache.ColorSourceCaching[frameIndex].IfNullThenLet(() =>
```

Comment:
e có thể viết như này
DisplayedBitmapSourceCache.ColorSourceCaching?
    .Apply(it => it[frameIndex] = it[frameIndex]
        .IfNullThenLet(() => DisplayedBitmapSourceCache.DisplayedBitmapSource?
            .Let(it => this.CountColors(it))));